02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,63 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Matthew Carson</title>
 </head>
 <body>
-
+  <header>
+    <h1>Matthew Carson</h1>
+    <h3>Kinda weird but you'll like it</h3>
+    <img src="profile.jpg"alt="Matt Carson picture"/>
+  </header>
+  <section id="Bio">
+    <h2>Bio</h2>
+    <p>I'm an Atlanta native who loves comic books and action movies. I was in my highschool's first graduationg class and got a degree in Mechanical Engineering. I've been working for the past few years as an electrical engineer and I'm looking to make a change.</p>
+  </section>
+  <aside>
+    <ul>
+      <li>Height: 5'9"</li>
+      <li>Eye Color: Brown Eyes</li>
+      <li>Race: Black</li>
+      <li>Weight: 240lb</li>
+    </ul>
+  </aside>
+  <section id="My Media">
+    <article id="What I'm Watching">
+      <h3>What I'm Watching</h3>
+      <figure>
+        <img src="preacher.jpg" height="300" width="200"/>
+        <figcaption>Preacher</figcaption>
+      </figure>
+      <figure>
+        <img src="mr_robot.jpg" height="300" width="200"/>
+        <figcaption>Mr. Robot</figcaption>
+      </figure>
+      <figure>
+        <img src="Steven.jpg" height="300" width="200"/>
+        <figcaption>Steven Universe</figcaption>
+      </figure>
+    </article>
+    <article id="What I'm Reading">
+      <h3>What I'm Reading</h3>
+      <ol>
+        <li>Leviathan Wakes</li>
+        <li>The Brief Wondrous Life of Oscar Wao</li>
+        <li>Makers</li>
+      </ol>
+    </article>
+  </section>
+  <footer>
+    <h1>Contact Me</h1>
+    <p>Call me at <a href="tel:4044926317">(404)492-6317</a></p>
+    <p>Reach out on Social Media</p>
+    <ul>
+      <li><a href="https://www.facebook.com/matthew.carson.3386">Facebook</a></li>
```

Comment:
Adding target="_blank" to your <a> tags will make the link open in a new tab, allowing the user to stay on your site (which is what we want after all, right?!).